UPGRADING.rst

```diff
@@ -10,6 +10,38 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2143 Merge service accounts for indexer and service
+============================================================
+
+The following directions need to be followed for each deployment that needs to
+merge the commit.
+
+1.  Let someone who can administer authorization of SAM/TDR know of the desired
+    change in service account via Slack. Note performing these steps with out
```

Comment:
Be more specific. This isn't some ominous "change in the service account". The SA is being renamed. Explain what exactly does this person needs to do.